client/state/timezones/test/actions.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+
+/**
+ * Internal dependencies
+ */
+import {
+	timezonesRequestAction,
+	timezonesRequestSuccessAction,
+	timezonesReceiveAction,
+} from '../actions';
+
+/**
+ * Fixture data
+ */
+import {
+	ACTION_TIMEZONES_RECEIVE,
+	ACTION_TIMEZONES_REQUEST,
+	ACTION_TIMEZONES_REQUEST_SUCCESS,
+	TIMEZONES_DATA,
+} from './fixture';
+
+describe( 'actions', () => {
+	describe( 'creators functions', () => {
+		it( '#timezonesRequestAction()', () => {
+			expect( timezonesRequestAction() ).to.eql( ACTION_TIMEZONES_REQUEST );
```

Comment:
It doesn't look like we're reusing the action values. It'd be a bit easier to read if the data is inlined here then.